### ISSUE TITLE
IPv6: Address family for hostname not supported

### ISSUE THREAD

**Issue by grinapo** [akpw/mktxp#125]
```
2024-03-19 18:13:57 Connection to router adam@2001:123:4:f101::12 has failed: [Errno -9] Address family for hostname not supported
```

putting it in `/etc/hosts` also reveal that mktxp seems somehow force dns over the local resolver? (Or tries to get A instead of any.)

```
2024-03-19 18:19:42 Connection to router adam@adam.local.grin has failed: [Errno -2] Name or service not known
```

**Comment by grinapo** [comment]
Seems this one definitely required but it's a little hazy for me what are their _releases_, or whatever they do.

https://github.com/socialwifi/RouterOS-api/pull/88

**Comment by irrwitzer42** [comment]
I would love to see / need this, too. I'm running a 100% IPv6 network, so I don't have ipv4 addresses to connect to on most of my routers.

**Comment by irrwitzer42** [comment]
I applied the changes from the above mentioned commit (March 2023) manually to the pip-installed module of version 0.17.0 (Sep 2019):
https://github.com/socialwifi/RouterOS-api/pull/88/commits/2e18fa45dd76665b4c9cb577b625c6f3bc2c3f54

/usr/local/lib/python3.12/site-packages/routeros_api/api_socket.py

This did the trick. Since I'm using the dockerized version, I will just add this modification to the docker build process.

**Comment by jgoclawski** [comment]
https://github.com/socialwifi/RouterOS-api/pull/88 has just been released as version 0.18.

**Comment by akpw** [comment]
great, thanks @jgoclawski
